packages/bpk-component-rating/src/BpkRating-test.js

```diff
@@ -212,4 +212,16 @@ describe('BpkRating', () => {
     );
     expect(asFragment()).toMatchSnapshot();
   });
+
+  it('should correctly when value is string type', () => {
+    const { asFragment } = render(
+      <BpkRating
+        ariaLabel="4,6 Wonderful Wise choice"
+        title="Wonderful"
+        subtitle="Wise choice"
+        value="4,6"
```

Comment:
In some language, the decimal point might be `,` rather than `.` .Some online example in production. See [link](https://www.skyscanner.net/hotels/search?entity_id=27544008&checkin=2022-06-04&checkout=2022-06-05&adults=1&rooms=1&impression_id=b2702e51-e8ce-4a6b-880e-d9efa81dc5f8).
![image](https://user-images.githubusercontent.com/33623220/170976983-6d10f477-e1b5-4eb4-b11e-21d9cf68c83f.png)

Reply:
We are using `formatNumber` from `saddlebag-localisation` to localize numbers to localized number strings, so usually we have to pass a string as the value instead of passing a number